doc/api/configuration.md

```diff
@@ -241,3 +241,33 @@ print(res)
 res = %sql SELECT * FROM languages LIMIT 2
 print(res)
 ```
+
+## `persist_snippets`
+
+```{versionadded} 0.8.1
+```
+
+Default: `False`
+
+Stores the snippet in a SQL file.
+
+```{code-cell} ipython3
+%config SqlMagic.persist_snippets = True
+```
+
+```{code-cell} ipython3
+%%sql --save rated_high
```

Comment:
let's modify the example so it stores two snippets, where the second one depends on the first one

Reply:
Sure!

Reply:
made the change